lavamoat/build-system/policy.json

```diff
@@ -8739,4 +8739,4 @@
       }
     }
   }
-}
```

Comment:
Note for myself:
---
`@jest/reporters` was already in the tree (as a dependency of `@jest/core`), but making it a direct dependency changed its priority in yarn's resolution algorithm. This caused yarn to relabel the shared `@jridgewell/trace-mapping` module from being "owned by" `terser-webpack-plugin` to being "owned by" `@jest/reporters`.